internal/k8shandler/log-exploration-api_test.go

```diff
@@ -0,0 +1,71 @@
+package k8shandler
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift/cluster-logging-operator/test/matchers"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"testing"
+)
+
+var _ = Describe("fluentd.go#newFluentdPodSpec", func() {
```

Comment:
How does deployment of this container relate to fluentd?

Reply:
Updated string by replacing fluentd with exploration-api in pull request #1208.